python/src/acp_sdk/client/client.py

```diff
@@ -62,6 +62,11 @@ async def agent(self, *, name: AgentName) -> Agent:
         self._raise_error(response)
         return AgentReadResponse.model_validate(response.json())
 
+    async def provider_healthcheck(self) -> bool:
+        response = await self._client.get(f"/healthcheck")
+        self._raise_error(response)
+        return response.json() == "OK"
```

Comment:
The concept of provider doesn't really exist in ACP. There is just client and server. Better to just name this `ping` here on the client and `healthcheck` on the server.